packages/plugins/typescript/react-apollo/src/visitor.ts

```diff
@@ -513,4 +529,25 @@ export class ReactApolloVisitor extends ClientSideBaseVisitor<ReactApolloRawPlug
 
     return [mutationFn, component, hoc, hooks, resultType, mutationOptionsType, refetchFn].filter(a => a).join('\n');
   }
+
+  public get sdkContent(): string {
+    const sdkOperations = this._operationsToInclude.map(x => {
+      const optionalVariables =
+        !x.node.variableDefinitions ||
+        x.node.variableDefinitions.length === 0 ||
+        x.node.variableDefinitions.every(v => v.type.kind !== Kind.NON_NULL_TYPE || v.defaultValue);
+
+      const documentVariableName = this.getDocumentNodeVariable(x.node, x.documentVariableName);
+
+      const operationName = x.node.name.value;
+      return `${operationName}${x.operationType}(variables${optionalVariables ? '?' : ''}: ${
+        x.operationVariablesTypes
+      }) {
+          return client.query<${x.operationResultType}>({variables, query: ${documentVariableName})
+      }`;
+    });
+    return `export const getSdk = (client: ApolloClient<any>) => {
```

Comment:
```suggestion
    return `export const getSdk = (client: Apollo.ApolloClient<any>) => {
```

Reply:
Yeah, you have helpers for getting the correct identifier. Using ApolloClient might break in case of a missing import.

Reply:
Hi, I haven't actually seen this, tried looking into generic-sdk, where can I find an example?

Reply:
I've added a commit which does add `Apollo`,

I couldn't figure out the helper method you mentioned, let me know where I can look into that, and I will update.